### PR TITLE
CORE: View an individual article

### PR DESCRIPTION
**Pull request by phil-pickering**
Allows users to go to a new page to read an individual article

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticle } from "../utils/api";
+import Header from "./Header";
+import Comments from "./Comments";
+
+export default function Article() {
+  const [article, setArticle] = useState([]);
+  const { article_id } = useParams();
```
Comment (Harrison365) [inline src/components/Article.jsx:9]: Nice 👍

File: src/components/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  let formattedDate = "";
+
+  if (article.created_at) {
+    formattedDate = article.created_at.slice(0, 10);
+  } else
+    return (
+      <>
+        <Header />
+        <p>Loading article...</p>
+      </>
```
Comment (Harrison365) [inline src/components/Article.jsx:26]: I cant see your isLoading state here. Shouldn't you have one for for this? ^^^

File: documentation/all-articles.html [hunk 3]
```diff
@@ -57,7 +57,8 @@ <h1>Northcoders News 🗞</h1>
     <div>
       <h2>Components</h2>
       <ol>
-        <li><code>App</code> (grey) contains the entire app.</li>
+        <li><code>Home</code> (grey) contains the entire screen.</li>
+        <li><code>Header</code> (green) displays the title.</li>
```
Comment (Harrison365) [inline documentation/all-articles.html:61]: Not sure that you need to be doing this file???

File: src/components/Article.jsx (hunk 2 above)
Comment (phil-pickering) [reply src/components/Article.jsx:26]: It doesn't need one as the condition is only true when the article has loaded. I've throttled my connection and it appears to be working as expected 😄

File: documentation/all-articles.html (hunk 3 above)
Comment (phil-pickering) [reply documentation/all-articles.html:61]: It's my planning and design mock-up file. I do the design in HTML & CSS first (mostly). Then I can reuse that code in my .jsx files with a bit of tweaking.

**Pull request merged**